azure/init.sh

```diff
@@ -88,7 +88,7 @@ semanage port -a -t ssh_port_t -p tcp 3456
 # restart ssh service to be able to use the new port
 systemctl restart sshd
 
-BRANCH=$5
+BRANCH="fix-epel-repo-install"
```

Comment:
```suggestion
BRANCH=$5
```

This was to test the changes introduced by this branch, so reverting.